src/shillelagh/backends/apsw/vt.py

```diff
@@ -190,11 +190,16 @@ def get_all_bounds(
         column_name = column_names[column_index]
         column_type = columns[column_name]
 
-        # convert constraint to native Python type, then to DB specific type
-        constraint = type_map[column_type.type]().parse(constraint)
-        value = column_type.format(constraint)
-
-        all_bounds[column_name].add((operator, value))
+        if isinstance(constraint, set) and operator == Operator.EQ:
+            # See also https://rogerbinns.github.io/apsw/vtable.html#apsw.VTCursor.Filter
```

Comment:
nit but I would write this as:
```python
if isinstance(constraint, set) and operator is Operator.EQ:
  # See also https://rogerbinns.github.io/apsw/vtable.html#apsw.VTCursor.Filter
  constraint = [type_map[column_type.type]().parse(c) for c in constraint]
  value = tuple(column_type.format(c) for c in constraint)
  operator = Operator.IN
else:
  constraint = type_map[column_type.type]().parse(constraint)
  value = column_type.format(constraint)

 all_bounds[column_name].add((operator, value))
```

Reply:
That causes warnings because in the 'if' branch value is a tuple and in the 'else' branch it is a string.

Reply:
You can solve that by forward declaring the type
```python
value: Tuple | string # update type as needed
if isinstance(constraint, set) and operator is Operator.EQ:
  # See also https://rogerbinns.github.io/apsw/vtable.html#apsw.VTCursor.Filter
  constraint = [type_map[column_type.type]().parse(c) for c in constraint]
  value = tuple(column_type.format(c) for c in constraint)
  operator = Operator.IN
else:
  constraint = type_map[column_type.type]().parse(constraint)
  value = column_type.format(constraint)

 all_bounds[column_name].add((operator, value))
```

Reply:
`Any` is fine